packages/loader/container-loader/README.md

```diff
@@ -77,7 +77,9 @@ Once closed, container terminates connection to ordering service, and any local
 
 The "closed" state effectively means the container is disconnected forever and cannot be reconnected.
 
-Container can also be closed by runtime itself as result of some critical error. Critical errors can be internal (like violation in op ordering invariants), or external (file was deleted). Please see [Error Handling](#Error-handling) for more details
+If after some time a closed container is no longer needed, calling `Container.dispose()` will dispose the runtime resources.
+
+Container can also be closed and disposed by runtime itself as result of some critical error. Critical errors can be internal (like violation in op ordering invariants), or external (file was deleted). Please see [Error Handling](#Error-handling) for more details.
```

Comment:
Same question as above - why are we talking about close-then-dispose?  I do remember it makes sense in terms of "preserving existing behavior" but semantically why wouldn't you just dispose in this case going forward?

Reply:
@markfields Maybe there's a case where someone wants to `close` the container to get it into a safe reading mode, and then `dispose` once it's no longer needed.
Calling `close` then `dispose` will keep the same functionality as before.

Reply:
Right.  I think a tiny tweak of `closed and disposed` -> `closed and/or disposed` might help.  It's probably fine as-is to a casual observer, I may have read into it in a weird way because I was already thinking about it.